File: LeetCode/Easy/Q485.java
package LeetCode.Easy;

public class Q485 {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int flag = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 1) {
                flag++;
                count = Math.max(count, flag);
            } else {
                flag = 0;
            }
        }
        return count;
    }
}

File: LeetCode/Python/Easy/Q485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums) -> int:
        maxCount = 0
        count = 0
        for num in nums:
            if num == 1:
                count += 1
            else:
                count = 0

            maxCount = max(maxCount, count)
        return maxCount
